scripts/build.ts

```diff
@@ -215,11 +215,11 @@ log(chalk.bold(`\n  –– ${chalk.red('Romcal')} builder ––`));
       // Do not output index.ts on iife format
       // and only output locale.iife.js on iife format
       if (
-        (format !== 'iife' || !/\/index\.ts$/.exec(p)) &&
+        (format !== 'iife' || !/[\\/]index\.ts$/.exec(p)) &&
```

Comment:
@emagnier, we could use `node:path.sep` instead of `[\\/]`, not only in regexes, but only in logs. WDYT?

Reply:
Good idea. But do not spend too much time on a refactor here (except if something isn't working on Windows), as I started a refactor on that part too (among others, converting the codebase to a monorepo and refactoring the build scripts).

Reply:
Okay, then I won’t change this to `node:path.sep`.

> I started a refactor on that part too (among others, converting the codebase to a monorepo and refactoring the build scripts

Where can I find this work of yours? I couldn’t find anything related in your fork.

Regarding monorepo: which monorepo tool you want to use? I presume Yarn-based, although I prefer using [Nx](https://nx.dev/) monorepo. What will be are apps/libs, how do you want to split romcal?

Reply:
It's not pushed to GitHub yet, and it's progressing a bit slowly, as I don't have much free time to work on it currently...

FYI I use `pnpm` as it is faster than `npm` and `yarn`, and also use less disk space (especially when caching npm packages). About other tooling, I'm using `esbuild` & `tsup`, and I'm considering using `turborepo` to orchestrate and streamline all of that.

It also requires quite some code refactoring, and not everything is ready yet. At least one of the goal is to split the core romcal library and the data (calendar definitions, locales...). I hope to be able to share something very soon.

Reply:
Yeah, I have recently started using `pnpm` too, but not in all repos (for some stuff needs to be modified).

I have read about Turborepo, but I haven’t actually used it. I use Nx and I am quite happy with it. Have you ever used Nx?

Regarding splitting romcal into libs: how about this:
- core romcal library;
- GRC calendar definitions (`proper-of-times.ts`, `proper-of-saints.ts`);
- particular calendars;
- localisations.

I’ve been even thinking about changing the data format from TS/JS to JSON with JSON schema defined for all calendars and locales. That way, the data could be used by any other app in any language. (Something similar was suggested by [@Dev1an](https://github.com/Dev1an) in #43.) WDYT?

---

Anyway, we should move this conversation to discussions. :wink:

Reply:
I never used Nx yet, just read a bit on it. 
It's also currently on the radar to change the data format: make them easier to edit/improve (for contributors), and easier to consume (by romcal or any other tool).
More details soon, and yes in another place from here 😄

Reply:
Impatiently waiting to review your work. :wink: